packages/mc-html-template/lib/load-env.js

```diff
@@ -1,16 +1,53 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 const fs = require('fs');
 
 // Keep a reference to the loaded config so that requiring the module
 // again will result in returning the cached value.
 let loadedEnv;
 
-module.exports = configPath => {
-  if (loadedEnv) return loadedEnv;
+const START_OF_ENV_PLACEHODER = 'env:';
+const isEnvVariablesPlaceholder = valueOfEnvConfig =>
+  valueOfEnvConfig && valueOfEnvConfig.startsWith(START_OF_ENV_PLACEHODER);
```

Comment:
This will break if the value does not only contain a placeholder.

For example:

```json
{
  "mcApiUrl": "https://mc-api.${env:CTP_REGION}.${env:CTP_CLOUD_REGION}.commercetools.com"
}
```

Reply:
Yeah, could be useful. I didn't see the case yet apart from theoretically being nice.

1. We could achieve this by saying that the string is a template literal. Then we could provide the process.env as values. Then we however loose the `env:` prefix. 
2. We could use a RegExp to extract matches etc pp

Do we actually see supporting other things than `env:` in the future? Like inputs?

Reply:
> Do we actually see supporting other things than env: in the future? Like inputs?

Not sure, but I would stick to the syntax `${env:}` since it's also used by other tools (like Prisma).

A regex should do the trick yes in replacing the values.

Reply:
I am working on it. Please call me on my mobile when you don't hear from me for 2-3 hours. I might have jumped off the balcony running away from RegExps.